### PR TITLE
Feature/860 google analytics

### PR DESCRIPTION
**Pull request by cjhaddad**
# 💅🏼 What issue does this fix?
#860 

## 🍂 Before You Submit

* [ ] Did you write tests?
* [ ] Did you run Google Lighthouse and/or WebAIM (Wave) on UI components in your PR?
* [ ] Does your PR contain any required translations?
* [ ] Did you run `bundle exec rspec` from the root?
* [ ] Did you run `bundle exec rails rswag` from the root?
* [ ] Did you run `bundle exec rubocop` from the root?
* [x] Did you run `yarn lint` in `/client`?
* [x] Did you run `yarn test` in `/client`?
* [ ] Are your primary keys UUIDs on any new tables?

## 🛷 Deployment

* [ ] Data Migrations
* [ ] Schema Migrations
* [ ] Dependencies

## 🧳 Steps to test
pull down branch and basically navigate all over the place! go to google analytics and confirm you see your user info and page_path stats

### CODE REVIEW

File: client/src/App.js [hunk 1]
```diff
@@ -24,8 +24,18 @@ import { useAuthentication } from '_shared/_hooks/useAuthentication'
 const Routes = () => {
   const { t } = useTranslation()
   const isAuthenticated = useAuthentication()
+  const user = useSelector(state => state.user)
   let location = useLocation()
 
+  useEffect(() => {
+    if (!window.gtag) return
+    console.log(user.id)
```
Comment (katelovescode) [inline client/src/App.js:32]: Can we take out the console log here?

**Review by katelovescode — CHANGES_REQUESTED** [review]
Quick note about the console log, otherwise this looks good!

File: client/src/App.js (hunk 1 above)
Comment (cjhaddad) [reply client/src/App.js:32]: yes for sure!

**Comment by cjhaddad** [comment]
> Quick note about the console log, otherwise this looks good!

great! I will remove and then merge!

**Pull request merged**